### ISSUE TITLE
Cannot open migration file from terminal after generating with ecto.gen.migration

### ISSUE THREAD

**Issue by drl123** [KronicDeth/intellij-elixir#2391]
**Describe the bug**
After running `mix ecto.gen.migration filename`, the link to the migration file is shown in the terminal window, but clicking on the link to the file does not open the file in the editor.

**To Reproduce**
1. In terminal, type `mix ecto.gen.migration test`
2. Migration will be generated and display a link (properly highlighted)
   
![image](https://user-images.githubusercontent.com/568926/145690472-6d11944d-ae69-4972-853f-cd8cc5e150fa.png)

3. Clicking on the link does nothing.

**Expected behavior**
Clicking the migration file link from the terminal window should open the file in the editor

**Desktop:**
 - OS: macOS
 - Version 10.14.6

**Erlang:**
- Installer ASDF
- Version 24

**Elixir:**
- Installer ASDF
- Version 1.13.0-otp-24

**Plugin:**
- Version 12.0.1
- Rubymine 2021.3

**Comment by KronicDeth** [comment]
Are you in an umbrella project?  I noticed that some links don’t work for umbrella projects, but I thought it was due the base name of the file appearing in more than one place.

**Comment by drl123** [comment]
Nope, just a plain old Phoenix application.  FWIW it did not work in RM 2021.2 with v11 plugins (tried my other machine that had not updated).  I thought this used to work at some point, but I don't have any older configurations to try other than these.

**Comment by sh41** [comment]
Sorry for the long silence on this.

I read the plugin's console filters. Neither can match what `mix ecto.gen.migration` prints:

- `FileReferenceFilter` is compiled from the expression `$FILE_PATH$:$LINE$`, so the colon and the digits are required.
- `LiteralStackTraceFilter` only matches the `[file: ..., line: ...]` keyword list.

`* creating priv/repo/migrations/….exs` carries no line number in either shape, so the plugin has never produced a link for it — not in 12.0.1, not today. The recent console-filter work in #3925 doesn't change that, in case it looks relevant.

So I can't account for the highlight in your screenshot; it wasn't ours. The IDE's reworked terminal does recognise bare relative paths now (Ctrl-click, opens at line 1), but that engine postdates 2021.

Could you check on a current release whether the path is clickable, and whether you have the reworked terminal enabled? Leaving this open until then.
